app/workers/publishing_api_notifier.rb

```diff
@@ -5,8 +5,8 @@ def perform(edition_id, update_type = "normal")
     edition = Edition.find(edition_id)
```

Comment:
I'm not really a fan of the name "notifier". Both of these classes are doing more than notifying, they're updating content.